packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -6800,6 +6800,18 @@ class RepaintBoundary extends SingleChildRenderObjectWidget {
 /// semantics subtree is collected. If [ignoring] is true, pointer-related
```

Comment:
would it make sense to move the `{@template flutter.widgets.IgnorePointer.Semantics}` line to above the `## Semantics` header now that it's so big? e.g. consider how it looks in [SliverIgnorePointer](https://master-api.flutter.dev/flutter/widgets/SliverIgnorePointer-class.html).